muffnn/mlp/mlp_classifier.py

```diff
@@ -53,6 +53,13 @@ class MLPClassifier(MLPBaseEstimator, ClassifierMixin):
         If int, the random number generator seed. If RandomState instance,
         the random number generator itself. If None, then `np.random` will be
         used.
+    solver: an instance of a subclass of `tf.train.Optimizer`, optional
```

Comment:
nitpicking: there should be spaces around the ":" here and below (also `random_state` above, which I didn't notice before).  cf. https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt